notebooks/ml_fairness_explainability/inclusive_ml/labs/1_inclusive_ml.ipynb

```diff
@@ -114,7 +114,7 @@
     "gsutil mb $BUCKET\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.      --project=$PROJECT
</pre><p>i think a trailing <code>\</code>  is missing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>